commit 51a00928afe600bb5dcadaa59c6bf32b0b20cf8e (refs/changes/60/2860/3)
Author: Rui Loura <rui@joyent.com>
Date:   2017-10-26T16:16:11-04:00 (1 year, 11 months ago)
    
    TOOLS-1904 joyent-gerrit/docs/user/README.md should include details on squash step
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/docs/user/README.md b/docs/user/README.md
index a0f5313..389c0d0 100644
--- a/docs/user/README.md
+++ b/docs/user/README.md
@@ -196,8 +196,18 @@ change:
      create mode 100644 README.md
     Successfully rebased and updated refs/heads/master.
 
-And then I push that to the magical reference `refs/changes/12` (because this is
-a new PatchSet for Change 12):
+This will open a file that looks something like this:
+
+    pick 95a7c80 add initial README
+    pick f6eb4be forgot to add content
+
+Change `pick` to `squash` for the second commit like so:
+
+    pick 95a7c80 add initial README
+    squash f6eb4be forgot to add content
+
+Save and exit.  Then I push that to the magical reference `refs/changes/12`
+(because this is a new PatchSet for Change 12):
 
     dap@sharptooth my-playground $ git push cr HEAD:refs/changes/12
     Counting objects: 4, done.
